### DISCUSSION TITLE
How can we build permeability limited liver model in PKSIM

### DISCUSSION THREAD

**Discussion by NileshGaud** [Open-Systems-Pharmacology/Forum#879] [Questions & Problems] [answered]
Dear All,

Is there any option in PKSIM that can be used to build a permeability limited liver model. Most literature suggest using this model for drug transporter related PBPK model building and DDI predictions. Especially when the disposition is governed mainly by uptake and efflux transporters.

**Comment by StephanSchaller** [comment]
Hi Nilesh, PK-Sim is permeability-limited by default

**Comment by StephanSchaller** [comment]
See: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties

And: https://docs.open-systems-pharmacology.org/mechanistic-modeling-of-pharmacokinetics-and-dynamics/modeling-concepts/principles-of-pbpk-modeling#l-adme-logic-and-routes-of-administration

**Comment by teutonicod** [comment, marked as answer]
Hi @StephanSchaller and @NileshGaud ,

Thanks for the question and the discussion, since I think it is a very interesting topic. I am not sure I would define the model in PK-Sim, "permeability limited". From my understanding this type of definition come from other software, for which  the model can be based on one of the two following assumptions: 

- perfusion limited (and in this case from my understanding the distribution is mainly driven by Kp)
- permeability limited, for which the distribution is driven by permeabilities

From my understanding, it is difficult to compare the model, but if we really want to, I would say that in PK-Sim we do not make any of the aforementioned assumptions, but we use both a distribution model and a permeability model to define organ Kp and permeabilities which both drive the distribution of a compound. Which one is more relevant, depends on the molecule phys-chem parameters.
I would not define PK-Sim a permeability limited model, since I suppose you could have a parameter set for which the distribution may become limited by perfusion (although I have to say that I never tested the molecule parameter space to check if this is possible). 

Donato

**Comment by StephanSchaller** [comment]
@teutonicod, yes, all what you write is correct and I do agree and thank you for providing this additional detail. Having a case of limitation by perfusion of permeation in PK-Sim depends on the physchem, but structurally, PK-Sim ALWAYS considers permeation. Other PBPK models (often used in e.g. environmental tox) are BY STRUCTURE perfusion limited models. They do not have a concept of permeation and organ subcompartments.

**Comment by NileshGaud** [comment]
Hi @StephanSchaller and @teutonicod 

Thank you for providing the details it really helped me to understand the difference between the two models. 

Nilesh

**Comment by pburkat** [comment]
In the case of highly lipophilicity volatile anesthetics that are inhaled, tissue distribution would be perfusion limited I think.
